src/main/java/org/opentripplanner/client/OtpApiClient.java

```diff
@@ -141,7 +140,17 @@ private JsonNode sendRequest(String formattedQuery) throws IOException {
     }
     var jsonNode = mapper.readTree(response.getEntity().getContent());
 
-    LOG.debug("Received the following JSON: {}", jsonNode.toPrettyString());
+    LOG.trace("Received the following JSON: {}", jsonNode.toPrettyString());
     return jsonNode;
   }
+
+  private String planGpsOrPlaceParameter(String key, String valuePlaceId, Coordinate valueGps) {
```

Comment:
If fromPlace/toPlace have a method to convert themselves to a string, we don't need this anymore.